### PR TITLE
Fixed height top slot

### PR DESCRIPTION
**Pull request by regiskuckaertz**
Says Pat:

> we have the nod to remove 728x90s from the Tech section in the next week or so as an initial trial which will inform removing them from the site

This revives the fixed height top slot, only this time the height is fixed at 250px everywhere and the ad slot takes care of its own height.

cc @jbreckmckye @kelvin-chappell

### CODE REVIEW

**Comment by jbreckmckye** [comment]
I notice we also have an extant 940x230 size - what happens to them?

File: static/src/stylesheets/module/_adslot.scss [hunk 1]
```diff
@@ -161,6 +154,13 @@
         padding-bottom: $gs-row-height / 2;
         padding-top: $gs-row-height / 2;
 
+        /* Oh horrible code, I'm looking at you. One day, you'll disappear.
+           One day, I'll fucking rip you apart.
+         */
+        &.ad-slot--h250 {
+            height: 250px;
```
Comment (jbreckmckye) [inline static/src/stylesheets/module/_adslot.scss]: If the ad has a height of 230, are we happy showing an extra 20px gap underneath it? (AFAIK, the ad will just align to the top of its container)

**Comment by regiskuckaertz** [comment]
James sayz we must keep it

File: static/src/stylesheets/module/_adslot.scss (hunk 1 above)
Comment (regiskuckaertz) [inline static/src/stylesheets/module/_adslot.scss]: That's a very good question. As the intent is to eventually remove the top slot animation, I would say yes.

**Comment by jbreckmckye** [comment]
Do we need to add an extra condition to `sticky-nav-banner.js`?

**Comment by regiskuckaertz** [comment]
Yes, adding it. Thanks.

**Comment by jbreckmckye** [comment]
👍

**Pull request merged**